File: src/ddd/simple/service/message_module/SendMessageService.java
package ddd.simple.service.message_module;

import ddd.base.persistence.EntitySet;
import ddd.simple.service.base.BaseServiceInterface;
import ddd.simple.entity.message_module.SendMessage;

public interface SendMessageService extends BaseServiceInterface {
	public SendMessage saveSendMessage(SendMessage sendMessage) ;
	
	public int deleteSendMessage(Long sendMessageId) ;
	
	public SendMessage updateSendMessage(SendMessage sendMessage) ;
	
	public SendMessage findSendMessageById(Long sendMessageId) ;
	
	public EntitySet<SendMessage> findAllSendMessage() ;
 
}

File: WebContent/ddd/simple/view/message_module/js/sendMessage/SendMessageService.js

define(["app"], function (app) {
	app.factory("SendMessageService", function($resource) {
		
		var sendMessageService = $resource("../DD/SendMessage/:action", {});
		sendMessageService.saveSendMessage=function(sendMessage,sucesscb,errorcb)	{
			sendMessageService.save({action:"saveSendMessage"},{sendMessage:sendMessage},sucesscb,errorcb);
		};
		
		sendMessageService.deleteSendMessage=function(sendMessageId,sucesscb,errorcb)	{
			sendMessageService.save({action:"deleteSendMessage"},{sendMessageId:sendMessageId},sucesscb,errorcb);
		};
		
		sendMessageService.updateSendMessage=function(sendMessage,sucesscb,errorcb) {
			sendMessageService.save({action:"updateSendMessage"},{sendMessage:sendMessage},sucesscb,errorcb);
		};
		 
		sendMessageService.findSendMessageById=function(sendMessageId,sucesscb,errorcb) {
			sendMessageService.get({action:"findSendMessageById",sendMessageId:sendMessageId},sucesscb,errorcb);
		};
		
		sendMessageService.findAllSendMessage=function(sucesscb,errorcb) {
			sendMessageService.query({action:"findAllSendMessage"},sucesscb,errorcb);
		}
		return sendMessageService;
})});
